packages/node/src/indexer/project.service.ts

```diff
@@ -189,11 +204,12 @@ export class ProjectService {
       'chain',
       'specName',
       'genesisHash',
+      'startHeight',
       'chainId',
       'processedBlockCount',
       'lastFinalizedVerifiedHeight',
       'schemaMigrationCount',
-      'bestBlocks',
+      'unfinalizedBlocks',
     ] as const;
```

Comment:
should `bypassBlocks` be added to this list?

Reply:
Thanks @DeveloperInProgress , added